examples/math/cubic_splines.rs

```diff
@@ -151,9 +151,9 @@ struct Curve {
     inner: Option<CubicCurve<Vec2>>,
 }
 
-impl From<CubicCurve<Vec2>> for Curve {
-    fn from(value: CubicCurve<Vec2>) -> Self {
-        Self { inner: Some(value) }
+impl From<Option<CubicCurve<Vec2>>> for Curve {
```

Comment:
Shouldn't the From<CubicCurve> implementation still stay?

Reply:
I ended up deleting even this one and just making `Curve` a tuple struct for the sake of simplicity; the only reason this existed was to handle the old conversions out of curve constructors in the old version.